.travis.xacro_test.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
```

Comment:
using `ADDITIONAL_DEBS='ros-$ROS_DISTRO-rosbash'` did not work as it could not resolve `$ROS_DISTRO`....it worked with job specific full-qualified name `ADDITIONAL_DEBS='ros-indigo-rosbash', though...

from doku:
```
ADDITIONAL_DEBS (default: not set): More DEBs to be used. List the name of DEB(s delimitted by whitespace if multiple DEBs specified). Needs to be full-qualified Ubuntu package name. E.g.: "ros-indigo-roslint ros-indigo-gazebo-ros" (without quotation).
```